utils/post_utils.jsx

```diff
@@ -307,3 +307,94 @@ export function getLatestPostId(postIds) {
 
     return '';
 }
+
+export function createAriaLabelForPost(post, author, isFlagged, reactions, intl) {
+    const {formatMessage, formatTime} = intl;
+
+    let ariaLabel;
+    if (post.root_id) {
+        ariaLabel = formatMessage({
+            id: 'post.ariaLabel.replyMessage',
+            defaultMessage: '{authorName} at {timestamp} wrote a reply, {message}',
+        },
+        {
+            authorName: author,
+            timestamp: formatTime(post.create_at, {weekday: 'long', month: 'long', day: 'numeric'}),
+            message: post.message,
+        });
+    } else {
+        ariaLabel = formatMessage({
+            id: 'post.ariaLabel.message',
+            defaultMessage: '{authorName} at {timestamp} wrote, {message}',
```

Comment:
As per the spec it should be reading the **time** before the **date**, otherwise it doesn't make sense to say "at" if it's followed by a date.

Please change in all applicable places

Reply:
Okay I've fixed all these issues. The timestamp hiding was weird, looks like it came from a bad merge.